### ISSUE TITLE
Application crashes without any notice or log file generated - path with special characters

### ISSUE THREAD

**Issue by french7181** [DGFiP/Test-Compta-Demat#2]
Hello

I tried to submit a file to the x64 binary version that can be gathered from the "Direction Generale des Finances Publiques" website (http://www.economie.gouv.fr/dgfip/outil-test-des-fichiers-des-ecritures-comptables-fec). The file is a file that I generated using our (old) accounting system and some scripts.

In order to validate that those scripts are functional I generated a rather small file and submitted it to the application.
I use a flat file format and "|" separated values I generated from an xls spreadsheet in which my script push the values from our accounting system.

The application just stops working after a few seconds. In a windows command window the returrned code is just 0. Any idea of what might be wrong?  Here below are two lines (sorry for not providing more but I have to rename fields and values for the sake of privacy).

JournalCode|JournalLib|EcritureNum|EcritureDate|CompteNum|CompteLib|CompAuxNum|CompAuxLib|PieceRef|PieceDate|EcritureLib|Debit|Credit|EcritureLet|DateLet|ValidDate|MontantDevise|Devise
ACH|JOURNAL DES ACHATS|841001|20140227|440000|BLA BLA|||FA LM/6900435|20140209|BLA BLA 02/2014||407,41|||20140227||
ACH|JOURNAL DES ACHATS|841001|20140227|610200|LOCATION MATERIEL ROULANT|||FA LM/6900435|20140209|BLA BLA 02/2014|350,00||||20140227||
ACH|JOURNAL DES ACHATS|841001|20140227|445660|T.V.A. DEDUCT.S/BIENS ET SERV.|||FA LM/6900435|20140209|BLA BLA 02/2014|57,41||||20140227||

**Comment by pommedeterresautee** [comment]
Hi,

The last field is called `Idevise` and not `Devise` according to article A 47 A 1 of Book of Tax procedures.
I have no crash with the two lines you posted before or after fixing the field name. If you want deeper analysis please post the part of the file you know make the application crash.

Kind regards,
Michaël

**Comment by french7181** [comment]
Just to be sure, I used exactly those 4 lines and relaunched the test and the application still crashes.
There might be something wrong with my execution environment then.

Any known issue with Windows 8.1 Pro x64?
Any potential conflicts with installed programs, development environments or libraries?
I also did not use the default install path.

As a hint, in the application folder, I see that the /log/altoweb2.db file is updated but the err_init_.log remains empty. In the tmp folder, I also see that a file with the extension .entete is generated.

**Comment by pommedeterresautee** [comment]
Nope, it's compiled PERL. So no need of a configured PERL environment.
I use it on W8 64 bits like you.

BTW, Montantdevise have no cap to the D.

**Comment by pommedeterresautee** [comment]
Something interesting is to launch the app from command line so Error messages (channel 1) are print in the console.

**Comment by french7181** [comment]
Hello I corrected the name of the Montantdevise field.
I also have executed both the testeur.bat file and the init.exe directly from the command line but the command line gets to the invite for next command even before the Tkinter GUI is displayed. 

In that case, the outcome is the same: the application crashes without any message being displayed or any information being written to the log ...

**Comment by pommedeterresautee** [comment]
As you may have noticed, the shortcut of the application is setup with an environment.

Therefore to test the init.exe you need first to cd in the correct folder then call init.

```
Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. All rights reserved.

C:\Users\mbenesty>cd C:\Test_Compta_Demat\testeur
C:\Test_Compta_Demat\testeur>init.exe
```

Moreover, did you try uninstalling / reinstalling? Other Computers?

**Comment by french7181** [comment]
Ok, I think I got it now: I did not use the default installation folder at first.
I did two manipulations: I removed the application from my non default installation path and tried to resinstall it in that non-default path: the problem persisted.

I then removed it and installed it to the default one, and it now works.

Might be related to the length of the length of the default path name or to the presence to non standard char (such as "é" or "è") in the full path.

**Comment by pommedeterresautee** [comment]
@npotelle May be using default path should be added to the documentation of the application?

**Comment by npotelle** [comment]
Thanks for this report.
Yes, there is a problem when installation path has non standard characters.
I reopen this bug to not forget to solve it !
